gulp/styles.js

```diff
@@ -61,13 +61,16 @@ export function stylesAfterReplacementStream() {
 				config.dev.styles.stage :
 				3
 			),
-			preserve: true,
 			features: (
 				configValueDefined('config.dev.styles.features') ?
 				config.dev.styles.features :
 				{
-					'custom-media-queries': true,
-					'custom-properties': true,
+					'custom-media-queries': {
+						preserve: false
```

Comment:
Set `preserve` for `custom-media` set to `false`. This will stop `custom-media` like `@media (--content-query) {` from printing in additon to the computer output (e.g. `@media screen and (min-width: 48em) {`), making the built CSS AMP compatible.